Backpack-SwiftUI/NavigationTabGroup/Classes/BPKNavigationTabGroup.swift

```diff
@@ -0,0 +1,112 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import SwiftUI
+
+/// A Group of tabs that allows a single tab to be selected at a time.
+public struct BPKNavigationTabGroup: View {
+    private let tabs: [NavigationTabItem]
+    private let style: BPKNavigationTabStyle
+    private let onItemClick: (_ index: Int) -> Void
+    
+    @Binding private var selectedIndex: Int?
+    
+    public init(
+        tabs: [NavigationTabItem],
+        style: BPKNavigationTabStyle = .default,
+        selectedIndex: Binding<Int?>,
+        onItemClick: @escaping (_ index: Int) -> Void
+    ) {
+        self.tabs = tabs
+        self.style = style
+        self._selectedIndex = selectedIndex
+        self.onItemClick = onItemClick
+    }
+    
+    public var body: some View {
+        ScrollView(.horizontal, showsIndicators: false) {
+            HStack(spacing: .md) {
+                ForEach(Array(tabs.enumerated()), id: \.element) { index, item in
+                    tab(for: item, index: index)
+                }
+            }
+            .padding(1) // to account for chip outlines
+        }
+    }
+    
+    @ViewBuilder
+    private func tab(for tab: NavigationTabItem, index: Int) -> some View {
+        BPKNavigationTab(
+            tab.text,
+            icon: tab.icon,
+            selected: selectedIndex == index,
+            style: style
+        ) {
+            onItemClick(index)
+        }
+    }
+}
+
+public extension BPKNavigationTabGroup {
+    struct NavigationTabItem: Hashable {
```

Comment:
could you rename it to just `Item` its a nested type so we know it's a NavigationTabItem already